access/src/main/webapp/WEB-INF/jsp/fullRecord/aggregateRecord.jsp

```diff
@@ -174,5 +174,6 @@
 		</table>
 	</div>
 </c:if>
-
+<h2 class="full-metadata">Detailed Metadata</h2>
```

Comment:
The inclusion of this section should be conditional on there being a MODS datastream. You might need to modify `PermissionsHelper` to add a `hasDescriptionAccess` method like the other ones in that file.